common/utils.js

```diff
@@ -325,6 +325,27 @@ function isValidStellarContract(address) {
     return StellarSdk.StrKey.isValidContract(address);
 }
 
+/// Token Manager Types supported by ITS
+/// These are the standardized token manager implementations across all chains
+const tokenManagerTypes = {
+    INTERCHAIN_TOKEN: 0,
+    MINT_BURN_FROM: 1,
+    LOCK_UNLOCK: 2,
+    LOCK_UNLOCK_FEE: 3,
+    MINT_BURN: 4,
+};
+
+/**
+ * Validates if a token manager type is supported for Stellar link token operations.
+ * Only LOCK_UNLOCK and MINT_BURN token manager types are allowed for linking tokens on Stellar.
+ *
+ * @param {number} tokenManagerType - The token manager type to validate
+ * @returns {boolean} - True if the token manager type is valid for Stellar link token operations, false otherwise
+ */
+const isValidStellarLinkTokenManagerType = (tokenManagerType) => {
+    return tokenManagerType === tokenManagerTypes.LOCK_UNLOCK || tokenManagerType === tokenManagerTypes.MINT_BURN;
+};
```

Comment:
common utils shouldn't have chain specific helpers. Instead make this generic similar to the address encoding
```suggestion
const isValidLinkType = (chainType, tokenManagerType) => {
    return tokenManagerType === tokenManagerTypes.LOCK_UNLOCK || tokenManagerType === tokenManagerTypes.MINT_BURN;
};
```
provide the chainType. native type is invalid everywhere. only mint burn and lock unlock are valid outside of EVM chains. This can be called with the destination chain Type when doing linkToken, and similarly for registerCustomToken for source chain type